### ISSUE TITLE
Multiple Windows on different Workspaces

### ISSUE THREAD

**Issue by nbsoftware** [ForkIssues/Tracker#2145]
I want to open say 2 Fork windows, each one of them on a separate workspace but this still doesn't work.

Asmof, it was mentioned in the workspace issue: https://github.com/fork-dev/Tracker/issues/1565#issuecomment-1426683452

I'm beginning to understand that this might come from the fact that the workspace feature does remember the window positions hence it resets it when switching to a another workspace.

Is there any workaround?
Will this be ever fixed / implemented?

**Comment by MrSkwiggs** [comment]
Just stopping by to add a +1, I really wish we could open separate workspaces in separate windows 🙏

**Comment by andyfoster** [comment]
Yeah I came here to report this too. Seems like a major bug and really makes Fork a lot less useful for managing multiple projects. Very sad to see it's been over a year. 
Hopefully they can fix it soon

**Comment by thekarel** [comment]
If by workspace you mean virtual desktop, then on OSX at least you can open any repo in a new tab (CMD+T) and then drag the tab so it becomes an independent window. You can move the window wherever you like.

**Comment by andyfoster** [comment]
I mean the "Workspaces" button in Fork that lets you have a bunch of repos that all open together in the same window that you can name like "Work" (that might have 10 repos), "Other Project" (with 3 repos), "Personal Projects" (5 repos) etc.
I can open a new window in Fork but as soon as I choose a different Workspace, that new window closes itself and the first window changes Workspaces to the one I just chose.
I could open a new window and then manually open all the repos in there each time but this seems like the whole point of the Fork Workspace feature.

**Comment by markusloffler** [comment]
Same here, please fix this!
